controllers/imagecollector/imagecollector_controller.go

```diff
@@ -48,9 +48,10 @@ import (
 )
 
 var (
+	log                    = logf.Log.WithName("controller").WithValues("process", "imagecollector-controller")
 	scannerImage           = flag.String("scanner-image", "", "scanner image, empty value disables scan feature")
 	collectorImage         = flag.String("collector-image", "", "collector image, empty value disables collect feature")
-	log                    = logf.Log.WithName("controller").WithValues("process", "imagecollector-controller")
+	scannerSeverity        = flag.String("scanner-severity", "CRITICAL,HIGH", "list of severity levels to report for scanner")
```

Comment:
We will need some way to expose this to the helm chart as well

Reply:
good idea, added